### PR TITLE
chinese translation

### PR DESCRIPTION
**Pull request by wayne666**
just add chinese translation

### CODE REVIEW

**Comment by leandromoreira** [comment]
Hi @wayne666 thanks so much, I don't know any Chinese at all 😞 so I'm gonna more people able to review this pr so I can merge it, is that right for you?

**Comment by wayne666** [comment]
Of course sure, i will correct it if have some problems. Thanks

**Comment by wayne666** [comment]
> Hi @wayne666 thanks so much, I don't know any Chinese at all 😞 so I'm gonna wait more people to be able to review this pr so I can merge it, is that right for you?

Of course sure, i will correct it if have some problems. Thanks

File: README-cn.md [hunk 1]
```diff
@@ -0,0 +1,966 @@
...
+   */
+  //StreamingParams sp = {0};
+  //sp.copy_audio = 0;
+  //sp.copy_video = 0;
+  //sp.video_codec = "libvpx-vp9";
+  //sp.audio_codec = "libvorbis";
+  //sp.output_extension = ".webm";
+```
+
+> 现在，说实话，[这比我想象中的难](https://link.zhihu.com/?target=https%3A//github.com/leandromoreira/ffmpeg-libav-tutorial/pull/54)，我必须深入理解 [FFmpeg 命令行源码](https://link.zhihu.com/?target=https%3A//github.com/leandromoreira/ffmpeg-libav-tutorial/pull/54%23issuecomment-570746749)，并且做很多测试，我想我肯定搞错了一些细节，因为我为 h264 强制 `force-cfr` 才能工作，并且仍然能看见一些` warning 信息（强制帧类型（5）为真类型（3））`。
```
Comment (chien-yu) [inline README-cn.md:966]: I suggest we keep the original warning message.

因為我必須為`h264`強制`force-frc`才能運作，但還是會看見一些警告訊息像是 `warning messages (forced frame type (5) at 80 was changed to frame type (3))`。

**Review by chien-yu — APPROVED** [review]
LGTM. Thanks for the great translation.

File: README-cn.md (hunk 1 above)
Comment (leandromoreira) [reply README-cn.md:966]: is this message still happening?

Comment (chien-yu) [reply README-cn.md:966]: I actually didn't test it by self, will try.

**Review by leandromoreira — APPROVED** [review]

**Pull request merged**